src/opl.c

```
@@ -1280,21 +1298,30 @@ static void errorMessageHook()
 void setErrorMessageWithCode(int strId, int error)
 {
     snprintf(errorMessage, sizeof(errorMessage), _l(strId), error);
+    errorMessageStringId = strId;
     guiSetFrameHook(&errorMessageHook);
 }
 
 void setErrorMessage(int strId)
```

Cppcheck (2.21.0)

[style] The function 'setErrorMessage' is never used.

(unusedFunction)